tesseract/tesseract_visualization/src/ignition/tesseract_ignition_visualization.cpp

```diff
@@ -374,27 +361,26 @@ void TesseractIgnitionVisualization::plotToolPath(const tesseract_planning::Inst
         const auto* swp = pi->getWaypoint().cast_const<StateWaypoint>();
         assert(static_cast<long>(swp->joint_names.size()) == swp->position.size());
         tesseract_environment::EnvState::Ptr state = state_solver->getState(swp->joint_names, swp->position);
-        addAxis(entity_manager_, *link_msg, cnt, link_name, state->link_transforms[tip_link] * tcp, 1);
+        addAxis(entity_manager_, *link_msg, cnt, link_name, state->link_transforms[tip_link] * tcp, 0.05);
```

Comment:
Yea, I am in the process of creating a marker class to simplify extending the visualization capabilities which it will be exposed as part of that. Should have somethings soon.